distribution/ddf-common/src/main/resources/bin/setenv

```diff
@@ -65,3 +65,4 @@ export JAVA_MAX_MEM=4096M
 # export KARAF_BASE # Karaf base folder
 
 export KARAF_OPTS="-Dfile.encoding=UTF8 -Dddf.home=$DDF_HOME"
+export JAVA_OPTS=-"server -Xms128M -Xmx4096M -XX:+UnlockDiagnosticVMOptions -XX:+UnsyncloadClass"
```

Comment:
If we don't set this here the default JAVA_OPS in the karaf script will be used which includes `-Dcom.sun.management.jmxremote`. I just checked and this was fixed by the karaf upgrade so this change is no longer necessary.